docs/migration/v7-to-v8.md

```diff
@@ -0,0 +1,1955 @@
+# Upgrading from 7.x to 8.x
```

Comment:
Given this is a duplicate of what's in `MIGRATION.md`, shall we avoid having this here for now? We can add it later when we formally make the switch.

Reply:
sure